README_CICD_INTEGRATION.md

```diff
@@ -123,10 +123,10 @@ As a build artifact a docker image is wanted versioned with bb-gitversion.
 
 The example builds the docker image with the Dockerfile located in `docker/Dockerfile` and creates a tag generated by a previous step running `bb-gitversion`.
 
-Adapt the last section in the `pipeline.yaml` to match your needs. The `image-((bb_name))` will be called `image-bb-example` and is needed to define the target docker image. The name and docker namespace is configured in `config.yaml`. This image section does not need to be modified. But you might to modify the job called `Create docker image for ((bb_name)) and push it`.
+Adapt the last section in `ci/pipeline.yaml` to match your needs. The `image-((bb_name))` will be called `image-bb-example` and is needed to define the target docker image. The name and docker namespace is configured in `config.yaml`. This image section does not need to be modified. But you might to modify the job called `build-and-deploy-((bb_name))`.
```

Comment:
"But you might to modify..." --> "Instead you might modify..."

Reply:
done